src/test/resources/cucumber/features/integrationTests/github/publish-processing.feature

```diff
@@ -0,0 +1,44 @@
+@GitHubIntegrationTests @IntegrationTest
+Feature: parse and then publish processing given SAST XML results, findings should be open or update an issue in GitHub
+
+  @GitHubCreateIssuesByFilter
+  Scenario Outline: publish results with severity filters and check GitHub issues are getting open
```

Comment:
publish results with severity filters and check GitHub issues are getting open -> publish results with severity filters and verify  GitHub issues are getting opened.